src/common/settings.js

```diff
@@ -289,6 +289,13 @@ const options_list = {
         "default": ""
     },
 
+    "sync-combat-tracker": {
+        "title": "Sync D&D Beyond Encounter with VTT turn tracker",
+        "description": "Currently only supports roll 20.",
```

Comment:
I would make this also into a quick setting for the encounters page itself (see how popup.js checks for the page and adds custom stuff to the options, like the monster whispers option)
I'd change the title to : "Synchronize the Combat Tracker with the VTT" (maybe?)
Change description to : "Synchronizes D&D Beyond's Combat Tracker from your Encounters with the VTT's combat tracker (Roll20 only)"